Implement $mdm-link-history on JPA (#4648)
hapifhir/hapi-fhir: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/docs/server_jpa_mdm/mdm_operations.md

}
```

## Link History

Use the `$mdm-link-history` operation to request a list of historical entries for a given set of `goldenResourceId`s or `sourceResourceId`s.  Either parameter is optional but **at least one** must be provided.

> MDM link history is made possible by a back-end configuration that enables saving the historical entries to a new audit table in the database.  This feature is enabled by default.  Some clients may wish to leave this feature disabled in order to save disk space.  If you wish to use this operation you must ensure that you have **NOT** set the following system property to true:  `non-resource-db-history.disabled`.

Reviewer: suggestion: Positive language is clearer:
```suggestion
MDM link history is made possible by a back-end configuration that enables saving the historical entries to a new audit table in the database.  This feature is enabled by default.  Some clients may wish to leave this feature disabled in order to save disk space.
The `non-resource-db-history.disabled` system property disables this operation.
```
